src/utils/Cache.ts

```diff
@@ -0,0 +1,246 @@
+import { AxiosRequestConfig, AxiosResponse } from 'axios';
+
+export type CacheTargets = CacheTarget[];
+export enum CacheTarget {
+  CACHE_API = 'CACHE_API',
+  MEMORY = 'MEMORY',
+}
+export type CacheResponse = {
+  data: any;
+  status: number;
+  statusText: string;
+  headers: any;
+};
+
+export const CACHE_API_KEY = 'sentinelhub-v1';
```

Comment:
Nitpick: maybe this should be `sentinelhub-js-v1`?

Reply:
We could, but then we should also delete 'sentinelhub-v1'  from users' browsers?

Reply:
Ah. Nevermind then, resolving. :)